src/typegate/engine/Cargo.toml

```diff
@@ -68,6 +69,9 @@ bytes.workspace = true
 protobuf.workspace = true
 protobuf-json-mapping.workspace = true
 
+# python
+pyo3 = { workspace = true, features = ["extension-module"] }
```

Comment:
> extension-module
nice!

Reply:
By the way, which python version are you experimenting on this?
Is there a way to lock the Python version directly here?